### ISSUE TITLE
Results?

### ISSUE THREAD

**Issue by jijup** [Adarsh97/LidarRegistration#1]
Thanks for adding me to the repo.  I haven't built the code yet. will do that soon. 
Do you have any results on registration? If yes, please put it below as a reply.

**Comment by Adarsh97** [comment]
Hi,
Do you want to see the final output ? 
Can you specify the actual running time of the code which gave the output you sent to subhasree ma'am ?

**Comment by jijup** [comment]
Yes, please. I didn't find the output in your report. So just curious!
I assume, they might have done it using any standard software, e.g., cloud compare. Did you try registering it via cloudcompare?
Also, you may take a look at [sparseICP](https://github.com/OpenGP/sparseicp). I would be useful for your comparison.

**Comment by Adarsh97** [comment]
I added the build folder contain input and output (partial) . 
After registering 25 point cloud( 1400000 3D points) it taking too much time for next iteration.
We are currently running a system having 16GB RAM.
"sample0.pcd"  is the final output file.

**Comment by jijup** [comment]
Why 25 point clouds? Are you segmenting the larger data into small data for registration? I thought your algo. takes as inputs the MLS and ALS as whole. Anyways, I'll take a look at the partial result. Thanks.

**Comment by Adarsh97** [comment]
we segmented the data  set into 59 point clouds.
algo can take them both , but for testing we only gave one.

**Comment by jijup** [comment]
What are the inputs point clouds for sample0.pcd?

**Comment by Adarsh97** [comment]
yes we are registering only the MLS segments .

**Comment by Adarsh97** [comment]
Input was the 59 point clouds (in pcd format) obtained from the MLS data.

**Comment by Adarsh97** [comment]
A zipfile (sample0.zip) contain 26 point cloud output is added

**Comment by jijup** [comment]
Just wondering whether this algorithm works for registering a MLS segment with ALS segment? Registration of MLS with ALS is highly relevant due to the fact that these point clouds are acquired separately using different platforms, possibly with different settings. 

Also check out the following article from Sensors if you haven't gone through yet. "Registration of Laser Scanning Point Clouds: A Review".

**Comment by Adarsh97** [comment]
we didn't think about how to combine both, we thought that also work in the same way, anyway we will research about it and get back to you soon.

**Comment by jijup** [comment]
> A zipfile (sample0.zip) contain 26 point cloud output is added
What are inputs? sample1.pcd to sample26.pcd?

**Comment by jijup** [comment]
> we didn't think about how to combine both, we thought that also work in the same way, anyway we will research about it and get back to you soon.

If the algo works only for mobile lidar, that's also fine! Check if it works reasonable well for the given LiDAR.

**Comment by Adarsh97** [comment]
The input are sample0.pcd to sample58.pcd .
We created a function for measuring the similarity among the point clouds . So point clouds will be registered based on the output of that function.So we can't say first 26 point clouds got registered.
We can say most similar 26 point clouds got registered for sure.

**Comment by Adarsh97** [comment]
We are currently running the algorithm in a supercomputer (HPC,high performance cluster), So we will update you once we get the full output.
